src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/DefenceResponseNotificationService.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.cmc.claimstore.rpa;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.events.response.DefendantResponseEvent;
+import uk.gov.hmcts.cmc.claimstore.rpa.config.EmailProperties;
+import uk.gov.hmcts.cmc.claimstore.utils.DocumentNameUtils;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.email.EmailAttachment;
+import uk.gov.hmcts.cmc.email.EmailData;
+import uk.gov.hmcts.cmc.email.EmailService;
+import uk.gov.hmcts.cmc.rpa.mapper.DefenceResponseJsonMapper;
+
+import java.util.Collections;
+
+import static java.util.Objects.requireNonNull;
+
+@Service("rpa/defendant-response-notification-service")
+@ConditionalOnProperty(prefix = "feature_toggles", name = "emailToStaff")
+public class DefenceResponseNotificationService {
+
+    public static final String JSON_EXTENSION = ".json";
+
+    private final EmailService emailService;
+    private final EmailProperties emailProperties;
+    private final DefenceResponseJsonMapper jsonMapper;
+
+    @Autowired
+    public DefenceResponseNotificationService(
+        EmailService emailService,
+        EmailProperties emailProperties,
+        DefenceResponseJsonMapper jsonMapper
+    ) {
+        this.emailService = emailService;
+        this.emailProperties = emailProperties;
+        this.jsonMapper = jsonMapper;
+    }
+
+    @EventListener
+    public void notifyRobotics(DefendantResponseEvent event) {
+        requireNonNull(event);
+
+        EmailData emailData = prepareEmailData(event.getClaim());
+        emailService.sendEmail(emailProperties.getSender(), emailData);
+    }
+
+    private EmailData prepareEmailData(Claim claim) {
+
+        return new EmailData(emailProperties.getRecipient(),
+            "J defence response " + claim.getReferenceNumber(),
+            "",
+            Collections.singletonList(createDefenceResponseAttachment(claim))
```

Comment:
Don't we need to enclose PDF as well the same way as it happens for sealed claim? If not why we have different requirement?

Reply:
In this case there's no body message to show is what I've gathered from the requirement.

Reply:
no @singh-surjeet this is different
They asked if it was possible for us to also send the pdf for the sealed claim as whenever one fails they put it in a failure directory, seems a bit weird tbh, it hasn't been asked for here but it should be clarified